src/clr-core/common/utils/dom.ts

```diff
@@ -14,18 +14,3 @@ export function getElementWidthUnless(element: HTMLElement, unless: boolean) {
   }
   return '';
 }
-export function getTranslateForChromeRenderingBugUnless(unless: boolean) {
```

Comment:
Are these functions no longer used?

Seems we'd get some use out of the `toggleDisabledAttribute()` one. And I'm curious as to why `getTranslateForChromeRenderingBugUnless()` was in there and now not.

Reply:
The `toggleDisabledAttribute` I can add back but seems like it's not needed since we are using custom elements we can do `this.disabled = true|false;`. I think we usually will want to set properties internally so the attributes are reflected and both stay in sync.

The `getTranslateForChromeRenderingBugUnless` was being applied every time the instance was created and applying the same CSS style inline so I moved it to the SCSS file instead.

Reply:
I'm fine with getting rid of them. Happy about it, even. I just wanted to understand why they weren't being used. 😊